### ISSUE TITLE
Mismatched dims for transformer-type vis in loss calculation when model's in_channels=out_channels

### ISSUE THREAD

**Issue by YongchengYAO** [ShihaoZhaoZSH/LaVi-Bridge#18]
In the training script `t5_transformer.py`,  we have
```python
 model_pred = model_pred.chunk(2, dim=1)[0]
```
[here](https://github.com/ShihaoZhaoZSH/LaVi-Bridge/blob/5a725f379f153991b36f307e099de01500f364c2/train/t5_transformer.py#L181) to match the dimensions of `model_pred` and `noise` in 

```python
# Optimization
loss = F.mse_loss(model_pred.float(), noise.float(), reduction="mean")   
```

The script is compatible with models where the number of output channels is twice the number of input channels, such as the model `PixArt-alpha/PixArt-XL-2-512x512` in the sample script. We can find the in and out channel dims [here](https://huggingface.co/PixArt-alpha/PixArt-XL-2-512x512/blob/main/transformer/config.json) 

For other models (such as [Efficient-Large-Model/SANA1.5_1.6B_1024px_diffusers](https://huggingface.co/Efficient-Large-Model/SANA1.5_1.6B_1024px_diffusers/blob/main/transformer/config.json)) where in_channels=out_channels, how to adjust the training script. Shall we just ignore this line?
```python
 model_pred = model_pred.chunk(2, dim=1)[0]
```
